basics.sh

```diff
@@ -101,17 +100,6 @@
     && git config --global init.templateDir ~/.git-template \
     && pre-commit init-templatedir ~/.git-template -t pre-commit -t pre-push \
     && echo "\n-> pre-commit configured <-"
-
-    echo "\nIf you don't yet know magnet for OSX, you're wasting time resizing windows!"
-    echo "Here's the link: https://apps.apple.com/gb/app/magnet/id441258766?mt=12"
-    read -p "Do you want to open that link now? [y/n]" -t 5 REPLY
-    case $REPLY in
-        [yY])
-            open "https://apps.apple.com/gb/app/magnet/id441258766?mt=12"
-        ;;
-        *)
-        ;;
-    esac
```

Comment:
You really should be using magnet.  Everyone should be using magnet

Reply:
Agreed! However i think that requiring user input via the terminal might spook some people who aren't technically orientated who might need to run this setup script

Reply:
It times out after 5 seconds.